File: BD/Classe/Donne.php
<?php
/**
 *
 */
class Donne
{

  private $_NumJury;
  private $_NumGroupe;
  private $_IdNote;

  function __construct($NumJury,$NumGroupe,$idNote)
  {
    $this->_NumJury = $NumJury;
    $this->_NumGroupe = $NumGroupe;
    $this->_IdNote = $idNote;
  }

  public function get_NumJury(){
    return $this->_NumJury;
  }

  public function get_NumGroupe(){
    return $this->_NumGroupe;
  }

  public function get_IdNote(){
    return $this->_IdNote;
  }
}


 ?>

File: MyApplication/app/src/main/java/com/example/olivet/myapplication/Donne.java
package com.example.olivet.myapplication;

/**
 * Created by schultz on 16/01/18.
 */

public class Donne {

    private int NumJury;
    private int NumGroupe;
    private int idNote;

    // Constructeur
    public Donne(int numJ, int numG, int idN) {
        this.NumJury=numJ;
        this.NumGroupe=numG;
        this.idNote=idN;
    }

    public int getNumJury() {
        return NumJury;
    }

    public void setNumJury(int numJury) {
        NumJury = numJury;
    }

    public int getNumGroupe() {
        return NumGroupe;
    }

    public void setNumGroupe(int numGroupe) {
        NumGroupe = numGroupe;
    }

    public int getIdNote() {
        return idNote;
    }

    public void setIdNote(int idNote) {
        this.idNote = idNote;
    }
}
